tests/integration.rs

```diff
@@ -181,27 +177,11 @@ fn main() -> Result<(), Box<dyn Error>> {
         let integrator = RK4::new(dynamics);
 
         // Add EOPData
-        let eop = coordinates::coordinate_transformation::EOPData::from_epoch(current_epoch)
-            .unwrap_or_else(|e| {
-                eprintln!("Warning: Failed to get EOP data: {}. Using defaults.", e);
-                coordinates::coordinate_transformation::EOPData {
```

Comment:
Same here. Now the test fails if no data can be used. If we still wanted to use the default data we would now do
`.unwrap_or_else(EopData::default())`